src/util/IncognitoBrowser.cpp

```diff
@@ -30,6 +30,10 @@ QString getPrivateSwitch(const QString &browserExecutable)
         {"firefox-esr", "-private-window"},
         {"chromium", "-incognito"},
         {"brave", "-incognito"},
+        {"firefox-devedition", "-private-window"},
+        {"firefox-beta", "-private-window"},
+        {"firefox-nightly", "-private-window"},
+        {"qutebrowser", "--target", "private-window"},
```

Comment:
This opens a new window, not sure if there is a better version that opens a tab in an existing private window, might be worth noting.